Packs/CybleEvents/IncidentFields/incident_cybleeventsid.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
@cyble-dev Please use the common field "Event ID" instead.

Reply:
will go with the custom name as it would be easy to manage as one entity and tightly coupled

Reply:
used common field here